config/default/manager_auth_proxy_patch.yaml

```
@@ -51,4 +51,5 @@ spec:
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
         - "--catalogs-storage-dir=/var/cache/catalogs"
-        - "--feature-gates=CatalogMetadataAPI=true,HTTPServer=false"
+        - "--feature-gates=CatalogMetadataAPI=true,HTTPServer=true"
+        - "--http-external-address=https://core.catalogd-system.svc"
```

The test uses a different address. I'm guessing kustomize name prefixing is the culprit.

```suggestion
        - "--http-external-address=https://catalogd-core.catalogd-system.svc"
```

IMO, if we don't have one already, we should have a follow-up issue for making the kustomize stuff a bit more straightforward.

---

https://github.com/operator-framework/catalogd/issues/160